### DISCUSSION TITLE
not_equals constraint

### DISCUSSION THREAD

**Discussion by tlambert03** [pydantic/pydantic#3040] [Feature Request]
I have a local branch where I've implemented "not equals" for number types (e.g `conint(ne=0)`), with a JSON representation of `{not: {const: 0}}`.  Before I spend time putting together a PR, I'd like to know if this is something that would be considered.  thanks!

**Comment by edan-bainglass** [comment]
I just reported a similar feature request, then noticed yours. Not super hopeful considering yours has yet to see a response from pydantic.

**Reply by tlambert03** [reply]
I'll see if I can dig up that branch and submit it... see if that helps get attention

**Comment by edan-bainglass** [comment]
Sounds good. Please let me know if you get a response or action from pydantic's end. Thanks, and cheers 👍
